code/API_definitions/qod-api.yaml

```diff
@@ -329,18 +402,236 @@ components:
+        maxUpstreamRate:
+          allOf:
+            - $ref: "#/components/schemas/Rate"
+            - description: |
+              The maximum best effort data
+        maxUpstreamBurstRate:
+          allOf:
+            - $ref: "#/components/schemas/Rate"
+            - description: |
+              The burst rate for best effort data
+        targetMinDownstreamRate:
+          allOf:
+            - $ref: "#/components/schemas/Rate"
+            - description: |
+              This is the target minimum downstream rate for the QoS profile.
+
+              For 5G networks 3GPP Guaranteed Bit Rate (GBR) refers to a dedicated, fixed data rate assigned to
+              specific services, ensuring a minimum performance level. As per 3GPP TS 23.203,
+              GBR is a QoS parameter used to manage traffic classes in mobile networks. It
+              provides a stable data rate for latency-sensitive applications, such as voice calls or
+              video streaming, helping maintain a consistent user experience.  When this attribute is set
+              this could imply that a GBR QCI is used, though mechanisms on the network can be used to
+              ensure a minimum performance level without using a GBR QCI.
+
+              The committed data rate allocated to specific services, ensuring a consistent level of
+              performance. DOCSIS networks, the "Minimum Reserved Traffic Rate" is defined in the 
+              DOCSIS 3.1 - MAC and Upper Layer Protocols Interface Specification" 
+              and it ensures a consistent level of performance for specific services within the network.
+        maxDownstreamRate:
+          allOf:
+            - $ref: "#/components/schemas/Rate"
+            - description: |
+              The maximum best effort rate
+        maxDownstreamBurstRate:
+          allOf:
+            - $ref: "#/components/schemas/Rate"
+            - description: |
+              The maximum best effort burst rate
+        minDuration:
+          allOf:
+            - $ref: "#/components/schemas/Duration"
+            - description: |
+              The shortest time period that this profile can be deployed.
+        maxDuration:
+          allOf:
+            - $ref: "#/components/schemas/Duration"
+            - description: |
+              The maximum time period that this profile can be deployed.
+        priority:
+          type: number
+          example: 2.2
+          description: |
+            Priority levels allow efficient resource allocation and ensure optimal performance
+            for various services in each technology, with the highest priority traffic receiving
+            preferential treatment.
+            The lower value the higher priority.
+            Since not all access networks use the same priority range, so this priority will be
+            scaled to the access network's priority range.
+          format: float
+          minimum: 1
```

Comment:
When reviewing 3GPP QCI / 5QI definitions, I noted that:
- For 5QI, they multiplied the QCI priorities by 10, so priorities are now an integer < 100
- For QCI, there are priorities defined that are higher than (so numerically lower than) 1, specifically QCI 65 (0.7) and QCI 69 (0.5)

I'd suggest that `priority` is either:
- an integer between 1 and 99; or
- a float between 0 and 10

My preference is probably the latter, but either would work for 3GPP networks.